Look for gradle wrapper in build-root-directory

bigdaz opened the pull request:
This removes the need to specify `wrapper-directory` when specifying a non-default `build-root-directory`, and fixes #44.

This PR builds on top of #52.

eskatos commented on .github/workflows/prod.yml:
```diff
@@ -14,27 +14,14 @@ jobs:
+    - name: Test use Gradle wrapper
       uses: ./
       with:
-        wrapper-directory: __tests__/data/basic
```
:x: This removes test coverage for the `wrapper-directory` option. We should keep some coverage.

I understand those changes make this option unnecessary but it would be good to publish a `v1` update with this fix. I'd be more confortable doing this with some test coverage.

eskatos reviewed (changes requested):
Thanks again Daz, good stuff!
I requested some test coverage to be kept for `wrapper-directory`.

bigdaz commented:
I've cut down this PR to only include the single commit required to fix #44, and added back in the coverage for `wrapper-directory`. In the future I'll separate out any changes that are OK to add to `v1` from any breaking `v2` changes.

Having addressed the feedback, I'll merge optimistically.
